examples/db.yaml

```
@@ -13,31 +13,31 @@
-    """Create a new database"""
-    console.print(f"Creating database {name}", style="green")
-  delete|rm: |
-    """Delete a database"""
-    sure = typer.confirm("Are you really really really sure?")
-    if sure:
-        console.print(f"Deleting database {name}", style="red")
-    else:
-        console.print(f"Back to safety!", style="green")
+  create|mk:
+    args: &db_args
+    - name: DatabaseName
```

yamllint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)